integration/single/test_basic_api.py

```diff
@@ -113,3 +115,21 @@ def test_basic_api_with_tags(self):
         self.assertIsNotNone(stage)
         self.assertEqual(stage["tags"]["TagKey1"], "TagValue1")
         self.assertEqual(stage["tags"]["TagKey2"], "")
+
+    def test_state_machine_with_api_single_quotes_input(self):
```

Comment:
Another test without `UnescapeMappingTemplate` to show it's not desired?

Reply:
It's always desired AFAICS, but have to do as opt-in property to avoid large-scale redeployments.